plugins/settings.gradle.kts

```diff
@@ -0,0 +1,9 @@
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+        google()
+        gradlePluginPortal()
```

Comment:
```
* What went wrong:
Plugin [id: 'org.gradle.kotlin.kotlin-dsl', version: '2.1.7'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Included Builds (None of the included builds contain this plugin)
- Plugin Repositories (could not resolve plugin artifact 'org.gradle.kotlin.kotlin-dsl:org.gradle.kotlin.kotlin-dsl.gradle.plugin:2.1.7')
  Searched in the following repositories:
    Gradle Central Plugin Repository
```

@thunderbiscuit, adding `gradlePluginPortal()` repository here should fix the problem, though not entirely sure why 😅 

Usually, `plugins` adds Gradle plugin repository implicitly so adding this explicitly should not be necessary 🤔 🤷🏻‍♂️ 
> By default, the plugins {} DSL resolves plugins from the public [Gradle Plugin Portal.](https://plugins.gradle.org/)

Reply:
Actually, looks like Gradle plugin repository was down yesterday, so `gradlePluginPortal()` is in fact not necessary here, removing `gradlePluginPortal()`!